### PR TITLE
remove references to jenkins autoprovisioning

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@gabemontero ptal
@bmcelvee this probably needs some editing touches and possibly xref touchup but i hope it's a sufficient draft for you to carry it over the line.

The short version is we're dropping the jenkins auto provision feature, so:
1) references to that behavior and configuring that behavior need to be removed
2) we need to at least have minimal documentation on what users need to do so they know they need to provision their own jenkins and how to do it (the pipeline tutorial covers this).

**Comment by bparees** [comment]
also it looks like the 3.3 and 3.4 release notes linked to the file i'm removing.  Not sure the best way to resolve that.

File: dev_guide/openshift_pipeline.adoc [hunk 1]
```diff
@@ -20,19 +20,49 @@ Build Strategy, Jenkinsfiles, and the OpenShift Domain Specific Language (DSL)
 (provided by the OpenShift Jenkins Client Plug-in), you can create advanced
 build, test, deploy, and promote pipelines for any scenario.
 
+[[openshift-jenkins-sync-plugin]]
+== OpenShift Jenkins Sync Plugin
+
+This Jenkins plugin keeps OpenShift BuildConfig and Build objects in sync with Jenkins Jobs and Builds.
+
+The OpenShift jenkins Sync Plugin provides the following:
```
Comment (gabemontero) [inline dev_guide/openshift_pipeline.adoc]: jenkins -> Jenkins

File: dev_guide/openshift_pipeline.adoc [hunk 2]
```diff
@@ -20,19 +20,49 @@ Build Strategy, Jenkinsfiles, and the OpenShift Domain Specific Language (DSL)
...
+
+ * Dynamic job/run creation in Jenkins.
+ * Dynamic creation of slave pod templates from ImageStreams, ImageStreamTags, or
+ ConfigMaps.
+ * Injecting of environment variables.
+ * Pipeline visualization in the OpenShift web console.
+ * Integration with the Jenkins git plugin, which passes commit information from
+ OpenShift builds to the Jenkins git plugin.
```
Comment (gabemontero) [inline dev_guide/openshift_pipeline.adoc]: add secret to credential mapping

**Review by gabemontero — CHANGES_REQUESTED** [review]
just a couple of minor items

File: dev_guide/openshift_pipeline.adoc (hunk 1 above)
Comment (bparees) [reply dev_guide/openshift_pipeline.adoc]: fixed

File: dev_guide/openshift_pipeline.adoc (hunk 2 above)
Comment (bparees) [reply dev_guide/openshift_pipeline.adoc]: added

**Review by gabemontero — APPROVED** [review]
lgtm

**Comment by bparees** [comment]
@bmcelvee bump

File: architecture/core_concepts/builds_and_image_streams.adoc [hunk 3]
```diff
@@ -143,35 +143,14 @@ Pipeline workflows are defined in a Jenkinsfile,
 either embedded directly in the build configuration,
 or supplied in a Git repository and referenced by the build configuration.
 
-The first time a project defines a build configuration using a Pipeline
```
Comment (bmcelvee) [inline architecture/core_concepts/builds_and_image_streams.adoc]: I created the Pipeline build strategy module for 4.0 and removed the Jenkins autoprovisioning content in https://github.com/openshift/openshift-docs/pull/13000.

As I convert content, I'll continue updating this PR until it is complete in the 4.0 branch.

**Comment by bmcelvee** [comment]
@bparees @gabemontero We should be set to close this PR (*finally*). I modularized the content on the 4.0 branch, removed the autoprovisioning references, and added the Jenkins Sync Plugin.

**Comment by bmcelvee** [comment]
Closing per previous comment.

**Pull request closed without merging**